app/src/main/java/net/programmierecke/radiodroid2/PlayerService.java

```diff
@@ -922,7 +923,9 @@ public void foundLiveStreamInfo(StreamLiveInfo liveInfo) {
             }
         }
 
-        sendBroadCast(PLAYER_SERVICE_META_UPDATE);
-        updateNotification();
+        if (oldLiveInfo == null || !oldLiveInfo.getTitle().equals(liveInfo.getTitle())) {
```

Comment:
Maybe check all three significant fields of StreamLiveInfo?

Reply:
> Maybe check all three significant fields of StreamLiveInfo?

`title` is the only relevant It's split into "`artist` - `track`"